CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * Fix bug in parsing IPv6 addresses. Previously setting a target address or the
   initial upstream address in the config file would result in a malformed value.
   ([PR](https://github.com/hashicorp/boundary/pull/5221)).
+* ui: Fix text color when system mode is dark. ([PR](https://github.com/hashicorp/boundary-ui/pull/2651)).
```

Comment:
This bug fix doesn't need to be added since the only modal that had this issue was the hds::modal for "Discard unsaved changes?" that is going to be part of this release.

Reply:
Thanks ✨